### PR TITLE
adds mommi milk

### PR DESCRIPTION
**Pull request by toomykins**
mommis can now be milked for mommi milk
Screenshot_2021-01-21_11-07-42

:cl:
 * rscadd: added mommi milk

### CODE REVIEW

**Comment by ShiftyRail** [comment]
no

**Comment by toomykins** [comment]
:(

**Comment by Coxswain-Navigator** [comment]
But what does it DO?

File: code/modules/mob/living/silicon/mommi/mommi.dm [hunk 1]
```diff
@@ -317,4 +329,16 @@ They can only use one tool at a time, they can't choose modules, and they have 1
...
\ No newline at end of file
+	return
+
+/mob/living/silicon/robot/mommi/New()
+	udder = new(50)
+	udder.my_atom = src
+	..()
+
+/mob/living/silicon/robot/mommi/Life()
+	if(timestopped)
```
Comment (boy2mantwicethefam) [inline code/modules/mob/living/silicon/mommi/mommi.dm:340]: You probably won't need this if you were to do
`if(..())
   if(udder && prob(5))`

**Comment by jknpj** [comment]
Try vore, they might like it in their borgdogs.

**Comment by Senaat1** [comment]
MERGE THIS.

**Comment by ShiftyRail** [comment]
no

**Comment by Senaat1** [comment]
> no

Do it now. - Q

**Comment by boy2mantwicethefam** [comment]
LUMMOX HIRE THIS MAN

**Comment by xeroxemnas** [comment]
RE-OPEN THIS PR. THE PEOPLE DEMAND IT

**Comment by drawsstuff** [comment]
UGHHHHHHHHHHHHHHHHHHHHH MOMMIIIIIIIIIIIIIIIIII, GIMME GIMME ROBO MILKIESSSSSSSSSSSSSS PLEASEEEEEE!

**Comment by gurfan** [comment]
DOWN WITH THE COLLABS

**Comment by Eneocho** [comment]
Reminder that coders hate fun, apparently.

**Comment by xeroxemnas** [comment]
emojiocracy has spoken. REEEEEEEEOPEN IT

**Comment by cloutcode** [comment]
MERGE THIS

**Comment by Exxion** [comment]
You know, if you had opened this on April Fools it would have been speedmerged
Gotta be patient to make meme PRs

**Pull request closed without merging**